upload/extension/opencart/catalog/controller/module/carousel.php

```diff
@@ -16,8 +16,8 @@ public function index($setting) {
 		$results = $this->model_design_banner->getBanner($setting['banner_id']);
 
 		foreach ($results as $result) {
-			if (is_file(DIR_IMAGE . $result['image'])) {
-				$data['banners'][] = [
+			if (is_file(DIR_IMAGE . html_entity_decode($result['image'], ENT_QUOTES, 'UTF-8'))) {
+				$data['banners'][] = array(
```

Comment:
Change array to [] 53553cc57ec0e701dbbfcf77f04792b58bca76ca

Reply:
done

Reply:
#8269